src/ansys/templates/python/pyace_pckg/{{cookiecutter.__project_name_slug}}/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,64 @@
+# Contributor Covenant Code of Conduct
```

Comment:
Same as with the `CHANGELOG.md` file. This `CODE_OF_CONDUCT.md` should be included in the `templates/python/common/{{cookiecutter.__project_name_slug}}` so all templates ship by default with it.

Reply:
Same answer. I'mm move it